regolith/helpers/l_grantshelper.py

```diff
@@ -68,20 +66,39 @@ def sout(self):
         # print('rc.lead: {}'.format(rc.list))
         # print('rc.admin: {}'.format(rc.admin))
         grants = []
-        if rc.list and rc.admin:
+        if rc.list and rc.adm:
             raise RuntimeError(f"please specify either alias or admin, not both")
         for grant in self.gtx["grants"]:
             if rc.list and rc.list != grant.get('alias'):
                 continue
```

Comment:
This block also not needed, I won't ever filter on alias.  This means I don't need args for ``-l`` or ``-a``